project/publish-release.sh

```diff
@@ -77,9 +106,8 @@ echo ""
 echo ""
 echo "Git push to release branch"
 echo ""
-GIT_REPO="https://github.com/GoogleChrome/sw-testing-helpers.git"
 git init
-git remote add origin $GIT_REPO
+git remote add origin $GITHUB_REPO
 git checkout -b release
```

Comment:
If this is regarding grabbing from git or package.json - switched to get from git :)